battle-engine.js

```diff
@@ -1256,6 +1256,23 @@ BattlePokemon = (function () {
 		if (this.battle.gen >= 5) return ['Normal'];
 		return ['???'];
 	};
+	BattlePokemon.prototype.isGrounded = function () {
+		if (!this.hasType('Flying') && !this.hasAbility('levitate') && !this.hasItem('airballoon') && !this.volatiles['magnetrise'] && !this.volatiles['telekinesis']) {
```

Comment:
I don't think you should hardcode these without necessity, besides, this is broken in Inverse.